pyccel/epyccel.py

```diff
@@ -221,6 +221,11 @@ def epyccel( python_function_or_module, **kwargs ):
     comm  = kwargs.pop('comm', None)
     root  = kwargs.pop('root', 0)
     bcast = kwargs.pop('bcast', True)
+    if kwargs.pop('developer_mode', None):
+        # this will initialize the singleton ErrorsMode
```

Comment:
Great idea, just a small typo: this -> This.

Reply:
Haha, that's what happens when you copy/paste code. Fixed in 1d3c907